$NetBSD: patch-configure.ac,v 1.2 2015/06/14 16:08:04 wiz Exp $

Add a --with-shell option in order to avoid hardcoding "/bin/bash" inside
filter/foomaticrip.c.

Fixed upstream as [Bug 1288]:
https://bugs.linuxfoundation.org/show_bug.cgi?id=1288

--- configure.ac.orig	2015-04-13 14:47:41.000000000 +0000
+++ configure.ac
@@ -622,6 +622,16 @@ fi
 AC_SUBST(SED_EXTENDED_REGEX_OPT)
 AC_SUBST(CUPS_GHOSTSCRIPT)
 
+# =========================================================
+# Select a different shell instead of the default /bin/bash
+# =========================================================
+AC_ARG_WITH([shell],
+	[AS_HELP_STRING([--with-shell=path], [Specify path for a modern shell.])],
+	[with_shell="$withval"],
+	[with_shell="/bin/bash"]
+)
+AC_DEFINE_UNQUOTED([SHELL], "$with_shell", [Path for a modern shell])]
+
 # =====================
 # Prepare all .in files
 # =====================
@@ -659,6 +669,7 @@ Build configuration:
 	png:             ${with_png}
 	php:             ${with_php}
 	php-config:      ${with_php_config}
+	shell:           ${with_shell}
 	test-font:       ${with_test_font_path}
 	tiff:            ${with_tiff}
 	avahi:           ${enable_avahi}
